coalib/bearlib/languages/definitions/Java.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Java:
+    extensions = '.java'
+    comment_delimiter = '//'
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'"': '"', "'": "'"}
```

Comment:
as in from your comment earlier: 

chars != strings

they're used to find strings in your source code for whatever reason a bear wantt'